cloud/disk_manager/internal/pkg/dataplane/url/common/http_client.go

```diff
@@ -167,18 +167,16 @@ func (c *httpClient) head(
 		return nil, errors.NewRetriableError(err)
 	}
 
+	if resp.StatusCode == 403 {
+		return nil, NewSourceForbiddenError("http code %d", resp.StatusCode)
+	}
+
 	if resp.StatusCode >= 400 && resp.StatusCode <= 499 {
-		return nil, NewSourceNotFoundError(
-			"url source not found: code %d",
```

Comment:
'url source not found' is already printed there https://github.com/ydb-platform/nbs/blob/main/cloud/disk_manager/internal/pkg/dataplane/url/common/errors.go#L17